changelog/pending/20250501--programgen-go-nodejs-python--add-myinvokescalar-conformance-test.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: chore
+  scope: programgen/go,nodejs,python
+  description: Add myInvokeScalar conformance test
```

Comment:
Generally I would skip changelog for things like this -- it's not user facing, which is what we want the changelog to be.